api/v1beta2/alert_types.go

```diff
@@ -50,10 +50,12 @@ type AlertSpec struct {
 	// +optional
 	InclusionList []string `json:"inclusionList,omitempty"`
 
-	// EventMetadata is an optional field for adding metadata to events emitted by the
-	// controller. Metadata fields added by the controller have priority over the fields
-	// added here, and the fields added here have priority over fields originally present
-	// in the event.
+	// EventMetadata is an optional field for adding metadata to events dispatched by the
```

Comment:
Can we think of a way of wording this without getting into the detail of naming it "notification-controller"?

Reply:
"dispatching system"?

Reply:
Maybe even just using "controller" isn't that bad given the rest of the sentence has been restructured?

Reply:
fixed!